dev_scripts_helpers/documentation/render_images.py

```diff
@@ -348,9 +219,9 @@ def _render_image_code(
     # Remove the temp file.
     os.remove(in_code_file_path)
     # Update the cache.
-    if use_cache:
-        cache.update_cache(cache_key, cache_value)
-    return out_img_file_path, cache_hit
+    # if use_cache:
+    #     cache.update_cache(cache_key, cache_value)
```

Comment:
What about this?

If it's a leftover, one approach we use is to skim through the PR after sending it out to catch these small imperfections